.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+matrix:
+  fast_finish: true
+node_js:
+- '6'
```

Comment:
Since you use async/await, the nodejs version which should be used is 8. And this is also the version used in production.

Reply:
Can you remove your `request changes` since I updated the `travis.yml` configuration in the PR regarding configuration changes #3.